sympy/printing/pycode.py

```diff
@@ -94,6 +94,8 @@ class AbstractPythonCodePrinter(CodePrinter):
         fully_qualified_modules=True,
         contract=False,
         standard='python3',
+        num_terms_sum=42,
```

Comment:
honestly, 42 sounds pretty reasonable here, but it could also be just the actual limit such that the printer writes exactly the same code now as it did before except when that code would not be valid